cfme/automate/buttons.py

```diff
@@ -54,27 +53,6 @@
 ]
 
 
-class AutomateRadioGroup(RadioGroup):
```

Comment:
Are you sure it needs to be removed?
I see it's used in the same file in `class ButtonFormAnsibleView(View)`
`inventory = AutomateRadioGroup(locator=".//input[@name='inventory']/..")`

Reply:
ok, I see it's moved to ` widgetastic_manageiq/__init__.py`